### ISSUE TITLE
pypi.streamsx.hdfs: Provide composite class

### ISSUE THREAD

**Issue by anouri** [IBMStreams/pypi.streamsx.hdfs#9]
uses streamsx feature for pypi.streamsx.hdfs:
Support the concept of composite transformations.
IBMStreams/streamsx.topology#2322

**Comment by anouri** [comment]
pypi.streamsx.hdfs has been improved with 3 new classes to provide composite class feature.
```
HdfsDirectoryScan
HdfsFileSink 
HdfsFileSource
```
6 New test cases have been added to check the new feature.
```
 python3 -u -m unittest streamsx.hdfs.tests.test_hdfs.TestCompositeCloud
 python3 -u -m unittest streamsx.hdfs.tests.test_hdfs.TestCompositeDistributed
```

**Comment by anouri** [comment]
pypi.streamsx.hdfs has been improved with 4 new classes to provide composite class feature.

HdfsDirectoryScan
HdfsFileSink 
HdfsFileSource
HdfsFileCopy

8 New test cases have been added to check the new feature.

 python3 -u -m unittest streamsx.hdfs.tests.test_hdfs.TestCompositeWebHdfs
 python3 -u -m unittest streamsx.hdfs.tests.test_hdfs.TestCompositeDistributed

**Comment by anouri** [comment]
corrected in version 1.5.0
https://github.com/IBMStreams/pypi.streamsx.hdfs/releases/tag/v1.5.0
